src/components/plugins/PluginIndex.vue

```diff
@@ -64,7 +64,7 @@
     });
 
     const elementName = (qualifiedName: string) => {
-        let split = qualifiedName.split(".");
+        let split = qualifiedName.split(".");        
```

Comment:
Nitpick: prefer const to tell the compiler it can count on this being immutable.

```suggestion
        const split = qualifiedName.split(".");        
```